lib/drivers/oneTouchUltra2.js

```diff
@@ -0,0 +1,386 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2014, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+/**
+ * IOET's code start here
+ * pvalarezo@ioet.com
+ *
+ * The registers looks like:
+ * P "SAT","03/21/15","16:45:24   ","  081 ","N","03", 00 09B6
+ * P "SAT","03/21/15","16:42:11   ","  105 ","B","02", 00 099F
+ * P "SAT","03/21/15","16:39:46   ","  176 ","N","02", 00 09C1
+ *
+ */
+
+var _ = require('lodash');
+//Asyncronous javscript
+var async = require('async');
+//TidePool lib for ISO date handling
+var sundial = require('sundial');
+//TidePool struct lib
+var struct = require('../struct.js')();
+
+module.exports = function (config) {
+  var cfg = _.clone(config);
+  cfg.deviceData = null;
+  var serialDevice = config.deviceComms;
+
+  //BYTE START DEFINITION HEX
+  var DM = '\x11\x0d';
+  var DMDateTime = DM.concat('\x44\x4d\x46');
```

Comment:
For ease of debugging (looking at data dumps, etc) it's helpful not to use the hex codes for easily printable characters. 
I'd rather have you put this as 'DMF' instead of '\0x44\x4d\x46', and DM@, DMP, DMSU? and DMST? etc.